hphp/util/asm-x64.h

```diff
@@ -1205,6 +1205,11 @@ struct X64Assembler {
     bytes(n, nops[n]);
   }
 
+  void emitExceptions(int n) {
```

Comment:
"Exception" is a pretty overloaded term.  Can you rename this `emitTrap()` instead?  (And use the singular, to match `emitNop()`.